tests/common/test_transaction.py

```diff
@@ -1062,3 +1091,9 @@ def test_cant_add_empty_fulfillment():
         tx = Transaction(Transaction.CREATE, None)
     with raises(TypeError):
         tx.add_fulfillment(None)
+
+
+def validate(tx):
```

Comment:
This should probably be in a common.tests.util module, as `validate_transaction`